pkg/brokerapi/brokerapi.go

```diff
@@ -96,8 +96,10 @@ func (b BrokerAPI) Bind(ctx context.Context, instanceID, bindingID string, detai
 // Unbind deletes an existing service binding
 //   DELETE /v2/service_instances/{instance_id}/service_bindings/{binding_id}
 func (b BrokerAPI) Unbind(ctx context.Context, instanceID, bindingID string, details domain.UnbindDetails, asyncAllowed bool) (domain.UnbindSpec, error) {
-	spec := domain.UnbindSpec{}
-	return spec, errors.New("not implemented")
+	logger := requestScopedLogger(ctx, b.logger).WithData(lager.Data{"instance-id": instanceID, "binding-id": bindingID})
+	logger.Info("unbind-instance", lager.Data{"plan-id": details.PlanID, "service-id": details.ServiceID})
```

Comment:
```suggestion
	logger.Info("unbind-instance", lager.Data{"plan-id": details.PlanID, "service-id": details.ServiceID, "instance-id": instanceID, "binding-id": bindingID})
```

Reply:
Actually, I think at least the `instance-id` should be added to all logs in this file. Or is this logged somewhere else already?

Reply:
The instance-id is already in the `logger` (see line above) :) 

But logging is a topic I need to improve, along with the whole difference between brokerapi and internal/broker/broker. I leave that for a bit later :)